code-challenges.js

```diff
@@ -52,3 +149,30 @@ var numberExampleRefactor2 = "hello"
 
 var numberExampleRefactor3 = 27
 var numberExampleRefactor4 = 24
+
+// // Function Signature
+// // input: 2 numbers or a number and a string
+// // output: the difference of the smaller number from the larger number or error message
+
+// // Create a function called subtraction that takes in two numbers
+// const subtraction = (num1, num2) => {
+//     // if number 1 is larger than number 2
+//     if(num1 > num2) {
+//         // return the difference of number 2 from number 1
+//         return num1 - num2
+//     } 
+//     // if number 1 is less than number 2
+//     else if(num1 < num2) {
+//         // return the difference of number 1 from number 2
+//         return num2 - num1
+//     }
+//     // if number 1 or number 2 is not a number
+//     else if(typeof num1 != "number" || typeof num2 != "number")
+//         // return "Your input is not a number"
+//         return "Your input is not a number"
+// }
```

Comment:
Great use of specific evaluation. This is a good practice-- while a simple else statement would do the job here you are checking for datatypes. Wisdom here is knowing when this is needed (which is more often than devs generally want to do).